optaplanner-core/src/main/java/org/optaplanner/core/config/localsearch/decider/acceptor/AcceptorConfig.java

```diff
@@ -495,6 +525,31 @@ public Acceptor buildAcceptor(HeuristicConfigPolicy configPolicy) {
                     stepCountingHillClimbingSize_, stepCountingHillClimbingType_);
             acceptorList.add(acceptor);
         }
+        if ((acceptorTypeList!= null && acceptorTypeList.contains(AcceptorType.GREAT_DELUGE))
+                || greatDelugeRainSpeedRatio != null || greatDelugeRainSpeedScore != null) {
+            GreatDelugeAcceptor acceptor = new GreatDelugeAcceptor();
+
+            if (greatDelugeRainSpeedScore != null && greatDelugeRainSpeedRatio != null) {
+                throw new IllegalArgumentException("The greatDelugeRainSpeedScore (" + greatDelugeRainSpeedScore
+                        + ") and greatDelugeRainSpeedRatio (" + greatDelugeRainSpeedRatio
+                        + ") cannot be both non null.");
```

Comment:
Great work. One more thing to think of: can we set a default ratio? What would be a good default ratio? 0.05?

Motivation: SA is underused by our users because it requires a domain-specific parameter to be set.
LA and TS do not. They are used far more due to this, because they are easier to use.
Our users often do a blueprint benchmark and use the best one in production.
SA isn't blueprint friendly, so it's often not even considered by many users. That's a pity.
A good default ratio would make it blueprint-friendly, see docs "17.2.6. Benchmark Blueprint: A Predefined Configuration"

Reply:
The default ratio is here, incl. a comment for where the number comes from:
https://github.com/kiegroup/optaplanner/pull/513/files#diff-85928658d71f0163f269403e0d64432bR19

Did you mean something else?